### ISSUE TITLE
Fails to generate mask (Not Responding)

### ISSUE THREAD

**Issue by f0banzai** [KUR-creative/SickZil-Machine#100]
I've changed my PC from P106-100 (eq. GTX1060 6GB but TCC Mode only) to RTX3060 12GB LHR
I run SickZil，Import image，Then it is not responding.
CUDA Version 11.6, Driver version 511.23
Why? Why not responding?It ran well on P106-100.I don't know how to solve it.

**Comment by f0banzai** [comment]
OK I know.
DON'T USE AMPERE (RTX3000) because this version only supports cuDNN7 while Ampere GPUs support cuDNN 8 Lowest.

**Comment by Arenz2001** [comment]
Yeah, I have the same issue. But if you use the CPU version, It works pretty well. It's a good alternative for now
